eng/Tools.props

```diff
@@ -61,6 +63,36 @@
     <OnError ExecuteTargets="DumpInitToolsLog" />
   </Target>
 
+  <Target Name="InitGlobalTools" AfterTargets="Restore">
```

Comment:
Is it feasible/useful to put Inputs/Outputs on this to skip if everything seems up to date?

Reply:
It's not really useful to use input/outputs here as the `dotnet tool list` command tells you if an install  or no action is necessary.